packs/wix.js

```diff
@@ -0,0 +1,25 @@
+
+const UIStrings = {
+  /** Additional description of a Lighthouse audit that tells the user to optimize image formats, in the context of the Wix CMS platform. */
+  "modern-image-formats": "Wix automatically converts uploaded images to WebP—a modern image format that's optimized for web performance. When you encounter an image in a different format, check whether it is being served from a third party source. If so, consider uploading it from the Wix Media Manager instead. Find [more ways to optimize your site's media](https://support.wix.com/en/article/site-performance-optimizing-your-media).",
+  /** Additional description of a Lighthouse audit that tells the user to be aware of offscreen images, and low quality image placeholder behavior, in the context of the Wix CMS platform. */
+  "offscreen-images": "Images are automatically lazy loaded in most scenarios. Offscreen images may be downloaded below the viewport. These are very small, low-resolution placeholder images which are fast to load. As the user scrolls down the page, these placeholders are replaced by high-resolution images. This serves the same purpose as lazy loading. [Learn more](https://support.wix.com/en/article/site-performance-how-wix-improves-your-sites-loading-time#lazy-loading-images) about how Wix improves your site's loading time.",
+  /** Additional description of a Lighthouse audit that tells the user to defer loading of non-critical third-party libraries, in the context of the Wix CMS platform. */
+  "render-blocking-resources": "Wix ensures that our resources are not render-blocking. When [using third-party code](https://support.wix.com/en/article/site-performance-using-third-party-code-on-your-site) on your site, it’s important to add it in a non-blocking manner. For example, you can add defer or async attributes to your third-party script. You can also place the script at the end of the code body to ensure that your regular site content doesn’t get delayed or prevented from rendering.",
+  /** Additional description of a Lighthouse audit that tells the user how they can improve performance by avoiding animated images and using videos where possible, in the context of the Wix CMS platform. */
+  "efficient-animated-content": "Using GIFs can significantly slow down a page's loading speed. Where possible, use videos instead of GIFs to showcase animated content. Modern video encoding formats are both higher quality and smaller in size than GIF animations. [Learn more](https://support.wix.com/en/article/site-performance-optimizing-your-media#optimizing-videos-on-your-site) about optimizing your site's media.",
```

Comment:
This advice is not specific to wix sites. If there isn't some wix specific mechanism here then we should remove this string.